Adds compatibility for Extended Nether Backport and BYG

OddBirdStanley opened the pull request.

OddBirdStanley committed:
modified:   README.md
	modified:   build.gradle
	new file:   libs/LibraryEx.jar
	modified:   src/main/java/logictechcorp/netherex/NetherExConfig.java
	modified:   src/main/java/logictechcorp/netherex/handler/PlayerHandler.java
	modified:   src/main/java/logictechcorp/netherex/world/biome/NetherBiomeData.java
	modified:   src/main/java/logictechcorp/netherex/world/biome/NetherExBiomes.java
	modified:   src/main/resources/assets/netherex/lang/en_us.json
	modified:   src/main/resources/datapacks/bop_nether_biome_pack/data/netherex/nether_biomes/biomesoplenty/fungi_forest.json
	modified:   src/main/resources/datapacks/bop_nether_biome_pack/data/netherex/nether_biomes/biomesoplenty/glowstone_grotto.json
	modified:   src/main/resources/datapacks/bop_nether_biome_pack/data/netherex/nether_biomes/biomesoplenty/undergarden.json
	modified:   src/main/resources/datapacks/bop_nether_biome_pack/data/netherex/nether_biomes/biomesoplenty/visceral_heap.json
	new file:   src/main/resources/datapacks/byg_nether_biome_pack/data/netherex/nether_biomes/byg/embur_bog.json
	new file:   src/main/resources/datapacks/byg_nether_biome_pack/data/netherex/nether_biomes/byg/glowstonegardens.json
	new file:   src/main/resources/datapacks/byg_nether_biome_pack/data/netherex/nether_biomes/byg/sythiantorrids.json
	new file:   src/main/resources/datapacks/byg_nether_biome_pack/data/netherex/nether_biomes/byg/warpeddesert.json
	new file:   src/main/resources/datapacks/byg_nether_biome_pack/pack.mcmeta
	new file:   src/main/resources/datapacks/enb_nether_biome_pack/data/netherex/nether_biomes/extendednether/basalt_deltas.json
	new file:   src/main/resources/datapacks/enb_nether_biome_pack/data/netherex/nether_biomes/extendednether/crimson_forest.json
	new file:   src/main/resources/datapacks/enb_nether_biome_pack/data/netherex/nether_biomes/extendednether/soul_sand_valley.json
	new file:   src/main/resources/datapacks/enb_nether_biome_pack/data/netherex/nether_biomes/extendednether/warped_forest.json
	new file:   src/main/resources/datapacks/enb_nether_biome_pack/pack.mcmeta
	modified:   src/main/resources/datapacks/netherex_nether_biome_pack/data/netherex/nether_biomes/netherex/arctic_abyss.json
	modified:   src/main/resources/datapacks/netherex_nether_biome_pack/data/netherex/nether_biomes/netherex/fungi_forest.json
	modified:   src/main/resources/datapacks/netherex_nether_biome_pack/data/netherex/nether_biomes/netherex/ruthless_sands.json
	modified:   src/main/resources/datapacks/netherex_nether_biome_pack/data/netherex/nether_biomes/netherex/torrid_wasteland.json

OddBirdStanley committed:
Update README.md

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/LogicTechCorp/NetherEx?pullRequest=257) <br/>All committers have signed the CLA.

LogicTechCorp commented:
Can you remove the additional changes so the merge only contains the new biome packs and the config options to disable them?

OddBirdStanley committed:
Update README.md

OddBirdStanley committed:
Update build.gradle

OddBirdStanley committed:
deleted:    libs/LibraryEx.jar
	modified:   src/main/java/logictechcorp/netherex/NetherExConfig.java
	modified:   src/main/java/logictechcorp/netherex/handler/PlayerHandler.java
	modified:   src/main/resources/assets/netherex/lang/en_us.json

OddBirdStanley commented:
> Can you remove the additional changes so the merge only contains the new biome packs and the config options to disable them?

Done. The casting exception still has to be caught though.
